pom.xml

```diff
@@ -109,6 +110,12 @@
                 <artifactId>resteasy-spring-web</artifactId>
                 <version>${resteasy.ver}</version>
             </dependency>
+            <dependency>
```

Comment:
@Moicen please move this dependency into your own example module.

Reply:
ok